prowler/config/config.py

```diff
@@ -43,6 +44,19 @@
 config_yaml = f"{pathlib.Path(os.path.dirname(os.path.realpath(__file__)))}/config.yaml"
 
 
+def check_current_version(prowler_version):
+    try:
+        latest_version = requests.get(
+            "https://api.github.com/repos/prowler-cloud/prowler/tags"
+        ).json()[0]["name"]
+        if latest_version != prowler_version:
+            return f"(latest is {latest_version}, upgrade for the latest features)"
+        else:
+            return "(it is the latest version, yay!)"
+    except Exception:
+        return ""
```

Comment:
Maybe we need to log some error here and then return, like:
```suggestion
       log.error()
   finally:
       return
```